conf/landorpropertydisposal.routes

```diff
@@ -2,3 +2,7 @@ GET        /:srn/land-property-disposal/:index                 controllers.nonsi
 POST       /:srn/land-property-disposal/:index                 controllers.nonsipp.landorpropertydisposal.LandOrPropertyDisposalController.onSubmit(srn: Srn, index: Max5000, mode: Mode = NormalMode)
 GET        /:srn/change-land-property-disposal/:index          controllers.nonsipp.landorpropertydisposal.LandOrPropertyDisposalController.onPageLoad(srn: Srn, index: Max5000, mode: Mode = CheckMode)
 POST       /:srn/change-land-property-disposal/:index          controllers.nonsipp.landorpropertydisposal.LandOrPropertyDisposalController.onSubmit(srn: Srn, index: Max5000, mode: Mode = CheckMode)
+GET        /:srn/any-part-land-property-still-held/:index                 controllers.nonsipp.landorpropertydisposal.LandOrPropertyStillHeldController.onPageLoad(srn: Srn, index: Max5000, mode: Mode = NormalMode)
```

Comment:
Multiple disposals can be created for the same heldPropertyTransaction. Not sure if there is any need to reflect that in the URL, just highlighting it anyway, just for consideration.

Reply:
added a double index in the url, can't think of a cleaner solution as we need to propagate both positions when navigating between pages